Sources/Build/Command.link().swift

```diff
@@ -38,6 +38,10 @@ extension Command {
             args += ["-L\(prefix)"]
             args += ["-o", outpath]
 
+          #if os(OSX)
+            args += ["-F", try platformFrameworksPath()]
+          #endif
+
```

Comment:
This change is conflating two very different things. One is adding a new tool to our project, whereas this diff is adding support for importing `XCTest` to all targets. I am perfectly fine with both changes, but they shouldn't be in the same PR because the latter is a behavior change for all packages we support.